src/Oak/Cmd/Http/Conversion.purs

```diff
@@ -0,0 +1,65 @@
+module Oak.Cmd.Http.Conversion
+  ( defaultEncode
+  , defaultDecode
+  , makeEncoder
+  , makeDecoder
+  ) where
+
+import Data.Foreign.Class (class Decode, class Encode)
+import Data.Foreign.Generic (defaultOptions, genericDecode, genericDecodeJSON, genericEncode, genericEncodeJSON)
+import Data.Either (Either(..))
+import Data.Foreign.Generic (defaultOptions, genericDecode, genericDecodeJSON, genericEncode, genericEncodeJSON)
+import Data.Foreign.Generic.Class (class GenericEncode, class GenericDecode)
+import Data.Generic.Rep (class Generic)
+import Data.Foreign.Class (class Decode, class Encode)
+import Prelude (show, ($), class Show)
+import Control.Monad.Except (runExcept)
+import Data.Foreign (Foreign, F)
+import Data.Foreign.Generic.Types (Options)
+
+encodeOptions :: Options
```

Comment:
encodeOptions and decodeOptions seem to have the same type and value. Maybe just keep one or the other? Call it codingOptions or something?